osu.Game.Tests/Visual/UserInterface/TestSceneDeleteLocalScore.cs

```diff
@@ -158,14 +158,14 @@ public void TestDeleteViaRightClick()
                 InputManager.Click(MouseButton.Left);
             });
 
-            AddUntilStep("score removed from leaderboard", () => leaderboard.Scores.All(s => s.OnlineScoreID != scores[0].OnlineScoreID));
+            AddUntilStep("score removed from leaderboard", () => leaderboard.Scores.All(s => s != scores[0]));
```

Comment:
just out of curiosity, why was this required?

Reply:
Because they're sorted by total score (`ScoreInfo.TotalScore`).

Reply:
Right.. that's a weird test scene structure. Might change it while we're here.

Reply:
Has been fixed in https://github.com/ppy/osu/pull/14603/commits/d922210d2feda6f03de5717d3ca53e99b4864feb (was an actual issue, as discussed on discord).